solidity/contracts/bridge/Bridge.sol

```diff
@@ -26,7 +26,7 @@ import "./IRelay.sol";
 import "./BridgeState.sol";
 import "./Deposit.sol";
 import "./Sweep.sol";
-import "./Redeem.sol";
+import "./Redemption.sol";
 import "./BitcoinTx.sol";
```

Comment:
In the `BitcoinTx.sol` file we have a couple of types:
`Bitcon` -> `Bitcoin`.

Reply:
https://github.com/keep-network/tbtc-v2/pull/211/commits/1fac287a90313db582af388fc7062cdbb95375cc